fenix/app/src/main/java/org/mozilla/fenix/shopping/state/ReviewQualityCheckPreferences.kt

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.shopping.state
+
+import org.mozilla.fenix.utils.Settings
+
+/**
+ * Interface to get and set preferences for the review quality check feature.
+ */
+interface ReviewQualityCheckPreferences {
```

Comment:
is this intention here to abstract our `Settings` implementation away from the middleware?

Reply:
Yes